tests/network/checkup_framework/utils.py

```diff
@@ -149,6 +150,7 @@ def compose_configmap_data(
     dpdk_test_target_node=None,
     dpdk_vmgen_container_diskimage=None,
     dpdk_vmtest_container_diskimage=None,
+    verbose=None,
```

Comment:
Please use typing

Reply:
I'm changing verbose to be true always so I won't be touching the arguments.